src/main.cc

```diff
@@ -178,6 +178,25 @@ signal_cb(ev_loop *loop, struct ev_signal *w, int revents)
 	tarantool_exit(0);
 }
 
+static void
+signal_int_cb(ev_loop *loop, struct ev_signal *w, int revents)
+{
+    (void) loop;
+    (void) w;
+    (void) revents;
+
+    if (pid_file)
+        say_crit("got signal %d - %s", w->signum, strsignal(w->signum));
+
+    /* Regenerate the prompt on a newline. */
+    printf("\n");
```

Comment:
Using `printf()` from a signal handler is not safe. Prefer plain `write()`.

Reply:
Done.